feature/experimental/p4rt/otg_tests/traceroute_packetin_test/README.md

```diff
@@ -1,34 +1,124 @@
+
+Tests that traceroute respects transit FRR when the backup is also unviable.
+
+### Test-6
+
+Tests that traceroute respects decap.
+
+### Test-7
+
+Encap failure cases (TBD on confirmation)
+
+### Test-8
+
+Tests that traceroute for a packet with no route reports a miss.
 
+### Test-9
 
-## Config Parameter coverage
+Decap-And-Reencap cases (TBD)
 
-*    /components/component/integrated-circuit/config/node-id
-*    /interfaces/interface/config/id
+## Config Parameter Coverage
 
+*   network-instances/network-instance/name
+*   network-instances/network-instance/policy-forwarding/policies/policy/policy-id
+*   network-instances/network-instance/policy-forwarding/policies/policy/rules/rule/sequence-id
+*   network-instances/network-instance/policy-forwarding/policies/policy/rules/rule/ipv4/protocol
+*   network-instances/network-instance/policy-forwarding/policies/policy/rules/rule/ipv4/dscp-set
+*   network-instances/network-instance/policy-forwarding/policies/policy/rules/rule/ipv4/source-address
+*   network-instances/network-instance/policy-forwarding/policies/policy/rules/rule/ipv6/protocol
+*   network-instances/network-instance/policy-forwarding/policies/policy/rules/rule/ipv6/dscp-set
+*   network-instances/network-instance/policy-forwarding/policies/policy/rules/rule/ipv6/source-address
+*   network-instances/network-instance/policy-forwarding/policies/policy/rules/rule/action/decap-network-instance
+*   network-instances/network-instance/policy-forwarding/policies/policy/rules/rule/action/post-network-instance
+*   network-instances/network-instance/policy-forwarding/policies/policy/rules/rule/action/decap-fallback-network-instance
 
-## Telemetry Parameter coverage
+## Telemetry Parameter Coverage
 
-No new telemetry covered.
+*   network-instances/network-instance/name
+*   network-instances/network-instance/policy-forwarding/policies/policy/policy-id
+*   network-instances/network-instance/policy-forwarding/policies/policy/rules/rule/sequence-id
+*   network-instances/network-instance/policy-forwarding/policies/policy/rules/rule/ipv4/protocol
+*   network-instances/network-instance/policy-forwarding/policies/policy/rules/rule/ipv4/dscp-set
+*   network-instances/network-instance/policy-forwarding/policies/policy/rules/rule/ipv4/source-address
+*   network-instances/network-instance/policy-forwarding/policies/policy/rules/rule/ipv6/protocol
+*   network-instances/network-instance/policy-forwarding/policies/policy/rules/rule/ipv6/dscp-set
+*   network-instances/network-instance/policy-forwarding/policies/policy/rules/rule/ipv6/source-address
+*   network-instances/network-instance/policy-forwarding/policies/policy/rules/rule/action/decap-network-instance
+*   network-instances/network-instance/policy-forwarding/policies/policy/rules/rule/action/post-network-instance
+*   network-instances/network-instance/policy-forwarding/policies/policy/rules/rule/action/decap-fallback-network-instance
 
+## Protocol/RPC Parameter Coverage
```

Comment:
add p4rt RPCs as well?

Reply:
I don't think those are RPCs, right?

Reply:
they are,
https://github.com/p4lang/p4runtime/blob/d966be4591ecc0fba3a001872854fcac0a64052c/proto/p4/v1/p4runtime.proto#L29-L60

`SetForwardingPipelineConfig`, `StreamChannel`  and `Write` RPCs are involved in packetin scenarios AFAIR